docs/source-pytorch/advanced/model_parallel/index.rst

```diff
@@ -0,0 +1,102 @@
+
+Pre-training vs Fine-tuning
+===========================
+
+When fine-tuning, we often use a magnitude less data compared to pre-training a model. This is important when choosing a distributed strategy as usually for pre-training, **we are compute-bound**.
+This means we cannot sacrifice throughput as much as if we were fine-tuning, because in fine-tuning the data requirement is smaller.
+
+Overall:
+
+* When **fine-tuning** a model, use advanced memory efficient strategies such as :ref:`fully-sharded-training`, :ref:`deepspeed-zero-stage-3` or :ref:`deepspeed-zero-stage-3-offload`, allowing you to fine-tune larger models if you are limited on compute
+* When **pre-training** a model, use simpler optimizations such as :ref:`deepspeed-zero-stage-2`, scaling the number of GPUs to reach larger parameter sizes
+* For both fine-tuning and pre-training, use :ref:`deepspeed-activation-checkpointing` as the throughput degradation is not significant
+
+For example when using 128 GPUs, you can **pre-train** large 10 to 20 Billion parameter models using :ref:`deepspeed-zero-stage-2` without having to take a performance hit with more advanced optimized multi-gpu strategy.
+
+But for **fine-tuning** a model, you can reach 10 to 20 Billion parameter models using :ref:`deepspeed-zero-stage-3-offload` on a **single GPU**. This does come with a significant throughput hit, which needs to be weighed accordingly.
+
+When Shouldn't I use an Optimized Distributed Strategy?
+=======================================================
+
+Sharding techniques help when model sizes are fairly large; roughly 500M+ parameters is where we've seen benefits. However, in the following cases, we recommend sticking to ordinary distributed strategies
+
+* When your model is small (ResNet50 of around 80M Parameters), unless you are using unusually large batch sizes or inputs.
+* Due to high distributed communication between devices, if running on a slow network/interconnect, the training might be much slower than expected and then it's up to you to determince the tradeoff here.
+
+
+Cutting-edge and third-party Strategies
+=======================================
+
+Cutting-edge Lightning strategies are being developed by third-parties outside of Lightning.
+
+If you want to try some of the latest and greatest features for model-parallel training, check out the :doc:`Colossal-AI Strategy <../../integrations/strategies/colossalai>` integration.
+
+Another integration is :doc:`Bagua Strategy <../../integrations/strategies/bagua>`, deep learning training acceleration framework for PyTorch, with advanced distributed training algorithms and system optimizations.
+
+For training on unreliable mixed GPUs across the internet check out the :doc:`Hivemind Strategy <../../integrations/strategies/hivemind>` integration.
+
+
+----
+
+
+******
+Guides
+******
```

Comment:
This section is just a placeholder for navigation right now. The entire page here needs an overhaul.

Reply:
Shouldn't you also link to ddp_optimizations and model_init here for now?

Reply:
Yes, these will be nicely linked. Follow-up incoming shortly.